go.mod

```diff
@@ -32,9 +32,9 @@ require (
 	github.com/spacemeshos/economics v0.0.0-20220810190316-045462b489e8
 	github.com/spacemeshos/ed25519 v0.0.0-20220825090204-1f103b5756da
 	github.com/spacemeshos/fixed v0.0.0-20210523192743-8d17e03c169a
```

Comment:
are the changes in go.mod and go.sum related to the fixes? if not, can you break them into a separate PR?

Reply:
They are - Poet was upgraded.

Reply:
not a good deal. but how does poet upgrade affect these tests?

Reply:
There was a bug in Poet integration harness that caused it hang during shutdown. See https://github.com/spacemeshos/poet/pull/128.